data/docs.yml

```
@@ -53,6 +53,7 @@ analytics:
     - { title: 'libraries' }
     - { title: 'examples' }
     - { title: 'faq' }
+    - { title: 'new_analytics_tracking_script_migration_guide', url: '/analytics/new_script_migration_guide/' }
```

We don't need to add a URL. We just have to rename the title, to the name of the title.

```suggestion
    - { title: 'new_script_migration_guide' }
```